core/src/css/ionic.globals.ionic.scss

```
@@ -0,0 +1,209 @@
+// Global Font Functions
+@import "../themes/ionic.functions.font";
+
+$ionic-display-m:         dynamic-font(36px);
+$ionic-display-s:         dynamic-font(32px);
```

```suggestion
$ionic-display-s:         dynamic-font(32px);
$ionic-display-m:         dynamic-font(36px);
```

Sorry, nitpick, can we make these in the same order as the others 😬

---

No problem ;)